config/options.yml

```diff
@@ -28,3 +28,40 @@ rpm_repository:
     secret_key:
     bucket: rpm-manageiq-org
     endpoint: nyc3.digitaloceanspaces.com
+  :arches:
+    - "src"
+    - "noarch"
+    - "x86_64"
+  :content:
```

Comment:
Is this replaceable with config overrides?

Reply:
It can be overwritten by other people using it.  It would have to be in sync anywhere that we use it, otherwise it will cause problems.